### ISSUE TITLE
Release to jenkins-ci.org

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/jenkins.sitespeed.io#3]
https://wiki.jenkins-ci.org/display/JENKINS/Hosting+Plugins

**Comment by lukaswelte** [comment]
:+1:

**Comment by soulgalore** [comment]
yes, lets maybe should start checking this out next week. @lukaswelte have you done it before and have any advice?

**Comment by lukaswelte** [comment]
Sounds great.

Sorry, @soulgalore, I never did this before.

**Comment by soulgalore** [comment]
I've started the process now by joining the Jenkins mailing list, so hopefully it will be there in a week or so :)
